pyuvdata/uvdata/uvdata.py

```diff
@@ -5970,10 +5961,6 @@ def fix_phase(
                 "antenna positions. Please set use_ant_pos=True."
```

Comment:
@kartographer This error is currently being raised if you try to run fix_phase on a multi phase center data set regardless of the `use_ant_pos` parameter setting, but the message implies that setting that parameter to True will not raise the error. I started to add the conditional for that parameter, but then I realized that I'm not sure we want to run `fix_phase` on any multi phase center data set because those were almost certainly made with the new code. If that's the case I'll change the error message.

Reply:
It does indeed appear to be a bad rebase -- there used to be an `if use_ant_pos` above this in the logic. I can't immediately imagine a case where user should _want_ to do this, although my gut instinct is that it should just be a warning instead of an error (esp since it basically just calls `set_uvws_from_antenna_positions`). Lucky for us, `fix_phase` supplies such a warning now!

Reply:
fixed now.